README.md

```diff
@@ -1,83 +1,2 @@
-# HackIIITV 20 Submission
-This repository contains the rules and code of conduct for __HackIIITV 2020__.
```

Comment:
ye kyo uda diya bhai

Reply:
Sir Readme add kar rahe the toh file delete ho gayi. PR karne se pehle bhul gayi isko sahi karna. On it

Reply:
Sir done

Reply:
nice nice